models/widgets.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose')
+const { Schema } = mongoose
+
+const WidgetSchema = new Schema ({
+  type: String,
+  title: String,
+  size:  {type: Number, required: true},
+  contents: String
```

Comment:
This is something that we will address in our next branch/pull request. Will keep this feedback in mind!